partiql-types/src/main/java/org/partiql/types/PType.java

```diff
@@ -699,6 +711,15 @@ static PType sexp(@NotNull PType typeParam) {
         return new PTypeCollection(Kind.SEXP, typeParam);
     }
 
+    /**
+     * @return a PartiQL row type
+     * @deprecated this API is experimental and is subject to modification/deletion without prior notice.
+     */
+    @NotNull
+    static PType row(@NotNull Field... fields) {
+        return new PTypeRow(Arrays.asList(fields));
+    }
+
```

Comment:
self-review: auto-formatting messed with some things. All I've done is added the `row` helper.